src/handlers/gestures/panGesture.ts

```diff
@@ -13,7 +13,7 @@ export class PanGesture extends ContinousBaseGesture<PanGestureHandlerEventPaylo
     this.handlerName = 'PanGestureHandler';
   }
 
-  setActiveOffsetY(offset: number | number[]) {
+  activeOffsetY(offset: number | number[]) {
```

Comment:
This is unrelated.

Reply:
Yes, and I'm not sure how it ended up here, but since it is already on the `new-api` can we just leave it?